.github/workflows/ci.yml

```diff
@@ -4,6 +4,10 @@ on:
   push:
     branches:
       - master
+      - trying
```

Comment:
I don't think we need this commit at all because, as I mentioned in other comments, we aren't actually using bors-ng.

Reply:
Removed :)